### PR TITLE
clean-up and remove unnecessary code

### PR DESCRIPTION
**Pull request by guruathwal**
- Remove code for menu re-draw resulting in unnecessary code execution.
- Fix height of popup/dialog box without buttons.
- Cleanup code for displaying title text. 
- Misc. clean-up.

### CODE REVIEW

File: TFT/src/User/Menu/RRFMacros.c [hunk 1]
```diff
@@ -99,28 +98,26 @@ void menuCallMacro(void)
...
+          if (infoHost.connected != true)
+            break;
+
+          if (EnterDir(infoFile.longFile[key_num - infoFile.folderCount]) == false)
+            break;
+
+          runMacro(infoFile.file[key_num - infoFile.folderCount]);
+          update = 1;
+        }
+      break;
```
Comment (digant73) [inline TFT/src/User/Menu/RRFMacros.c:120]: wrong indentation

File: TFT/src/User/Menu/Popup.c [hunk 2]
```diff
@@ -208,28 +208,28 @@ void showDialog(DIALOG_TYPE type, void (*ok_action)(), void (*cancel_action)(),
...
 
-  // display the last received popup message, overriding previous popup messages, if any
-  if (popup_cancel[0])
+  if (popup_cancel[0]) // show both ok and cancel buttons
   {
     popupDrawPage(popup_type, bottomDoubleBtn, popup_title, popup_msg, popup_ok, popup_cancel);
     cur_btn_rect = doubleBtnRect;
   }
-  else if (popup_ok[0])
+  else if (popup_ok[0]) // show only ok button
```
Comment (digant73) [inline TFT/src/User/Menu/Popup.c:225]: two spaces for comment

File: TFT/src/User/Menu/ScreenSettings.c [hunk 3]
```diff
@@ -265,12 +264,11 @@ void menuLanguage(void)
       {
         totalItems[curItem].icon = CHARICON_UNCHECKED;
         listViewRefreshItem(curItem);  // refresh unchecked status
-        curItem = curIndex;
-        totalItems[curItem].icon = CHARICON_CHECKED;
-        listViewRefreshItem(curItem);  // refresh checked status
+        totalItems[curIndex].icon = CHARICON_CHECKED;
+        listViewRefreshItem(curIndex);  // refresh checked status
 
+        curItem = curIndex; // update selected index
```
Comment (digant73) [inline TFT/src/User/Menu/ScreenSettings.c:270]: two spaces for comment

**Comment by guruathwal** [comment]
Ok, I will look into it.

**Comment by digant73** [comment]
yes thanks, a check is always welcome. Although I fixed it in my PR #2419 (see my edit in my previous post).

**Pull request closed without merging**